apps/linode-marketplace-valkey/roles/post/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+# motd and cred dump
+
+- name: copy MOTD template to /etc/motd
+  template:
+    src: templates/motd.j2
+    dest: /etc/motd
+
+- name: writing deployment creds into file
+  copy:
+    dest: /home/{{ username }}/.credentials
+    mode: '0600'
+    owner: "{{ username }}"
+    group: "{{ username }}"
+    content: |
+      Sudo Username: {{ username }}
+      Sudo Password: {{ password }}
+      Valkey Password: {{ valkey_password }}
```

Comment:
This variable appears undefined. Did you mean `valkey_default_user_password` instead?